routes/api/category-routes.js

```diff
@@ -1,28 +1,76 @@
-const router = require('express').Router();
-const { Category, Product } = require('../../models');
+const router = require("express").Router();
+const { Category, Product } = require("../../models");
+const { findAll } = require("../../models/Product");
 
 // The `/api/categories` endpoint
 
-router.get('/', (req, res) => {
+router.get("/", async (req, res) => {
   // find all categories
-  // be sure to include its associated Products
+  // include its associated Products
+  try {
+    const categories = await Category.findAll();
```

Comment:
consider adding logic for some additional cases here e.g. if there are no categories present e.g.
```
    if (!categories.length) {
      return res.status(404).json({
        error: "No categories found"
      });
    }
```